.changie.yaml

```diff
@@ -28,7 +28,7 @@ custom:
   type: int
   minLength: 4
 footerFormat: |
-  Contributors:
+  ### Contributors
```

Comment:
This one is a no-look pass, I could be wildly off base

Reply:
It's easy to test all your changes work as expected!

Run this to print out what the generated yaml file will be:

```changie batch 1.1.0 --dry-run --include '1.1.0'```

`--dry-run` just prints out what would be generated and doesn't actually create a file or delete any yaml and `--include` will include the yaml files in that directory.

Note: Two new changes have been merged in since the release (2 files exists in `.changes/unreleased`) so those will be erroneously included.  Since it's not creating the files it doesn't matter.  This command should be enough to eyeball that the file is generated like you expect.

Reply:
realized I needed to update my installed version of `changie`! after that, works like a charm